client/rest-high-level/src/main/java/org/elasticsearch/client/ml/job/process/DataCounts.java

```diff
@@ -138,31 +145,13 @@ public DataCounts(String jobId, long processedRecordCount, long processedFieldCo
         this.lastDataTimeStamp = lastDataTimeStamp;
         this.latestEmptyBucketTimeStamp = latestEmptyBucketTimeStamp;
         this.latestSparseBucketTimeStamp = latestSparseBucketTimeStamp;
+        this.logTime = logTime == null ? null : Instant.ofEpochMilli(logTime.toEpochMilli());
```

Comment:
unsure the rounding to epoch ms this is really needed. The only time that users will be using this object with actual data is when it is deserialized from the server.

Reply:
We need it for tests. We use `Instant.now()` in tests and that includes nanos. We do that everywhere we use `Instant`.